e2e/parameter-type-extension/src/ParameterTypeExtension.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
```

Comment:
I'm not sure what the e2e/parameter-type-extension directory is for. These directories are supposed to execute something in `.github/workflows/e2e-tests.yml` but this one does not. Either move this under `e2e/bug-12585/` if it's used there, or delete it.